.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -311,6 +318,7 @@ def __call__(self, parser, namespace, values, option_string=None):
 
     SYS_ADMIN_EMAIL = get_input("What is the email for system administration?", defaults['sys_admin_email'])
     SYS_ADMIN_URL = get_input("Where to report problems with Submitty (url for help link)?", defaults['sys_admin_url'])
+    UPGRADE_LINUX_PACKAGES = get_input("Should Linux packages be updated with Submitty install?", defaults['upgrade_linux_packages'])
```

Comment:
I think this should say something like "Should Linux packages be automatically updated with each Submitty update install" to emphasize this is not a 1 time thing

Reply:
Thanks for pointing that, the text has been tweaked.